docs/book/src/topics/rosa/support.md

```diff
@@ -0,0 +1,17 @@
+# Create issue for ROSA
+
+When creating issue for ROSA-HCP cluster, include the logs for the capa-controller-manager and capi-controller-manager deployment pods. The logs can be saved to text file using the commands below. As well as, include the CRs that is used to create the ROSA ControlPlane, MachinePools and Cluster.
```

Comment:
```suggestion
When creating issue for ROSA-HCP cluster, include the logs for the capa-controller-manager and capi-controller-manager deployment pods. The logs can be saved to text file using the commands below.
Please also include the yaml for all the resources used to create the ROSA cluster:
- `Cluster`
- `ROSAControlPlane`
- `MachinePool`
- `ROSAMachinePool`
```
we need to explicitly list the resource names we want.

Reply:
okay